src/Linear/Annotations.tsx

```diff
@@ -182,12 +182,24 @@ const SingleNamedElement = (props: {
     }
   }
 
-  // determine whether the element name fits within the width of the element
-  const nameLength = name.length * 6.75; // aspect ratio of roboto mono is ~0.66
-  const nameFits = nameLength < width - 15;
+  // 0.591 is our best approximation of Roboto Mono's aspect ratio (width / height).
+  const annotationCharacterWidth = 0.591 * 12;
+  const availableCharacters = Math.floor((width - 30) / annotationCharacterWidth);
+  // Ellipsize the name if it's too long.
+  let displayName;
+  if (name.length <= availableCharacters) {
+    displayName = name;
+  } else if (availableCharacters <= 3) {
```

Comment:
@zeptonaut - Am I reading this right that if the `availableCharacters` = 4, the displayed name would be "N..." where only the first character of the name is displayed with an ellipsis?

If that's correct, that's probably not very helpful, even with a hover-over tooltip. Can we increase the threshold to 6 or 7 so that minimally 3 or 4 characters of the name are displayed?